frontend/src/routes/HomePage.js

```diff
@@ -106,7 +106,7 @@ class HomePage extends React.Component<{
                     <Grid item xs={12} className={classes.inner} container>
                         {/** Summary. */}
                         <Grid item xs={12} className={classNames(classes.textBox, classes.spacer)}>
-                            <Typography variant='h5' color='textSecondary' textAlign='center'>
+                            <Typography variant='h5' color='textSecondary' align='center'>
```

Comment:
The makes the text align differently from the current homepage (second image is current)
![Screenshot from 2019-11-13 10-00-11](https://user-images.githubusercontent.com/23012631/68790618-76211b00-05fc-11ea-8022-1cb7849640cb.png)
![Screenshot from 2019-11-13 10-00-19](https://user-images.githubusercontent.com/23012631/68790619-76211b00-05fc-11ea-86dc-772c937d7f8f.png)

Reply:
Good point. I assume that was the intended look and feel though when the frontend was first designed, since otherwise it wouldn't have used the `text-align` prop. If we want to keep it the same look, though, I can just go ahead and remove the `align` prop.

thoughts @percyliang ?

Reply:
I don't have a strong preference.  Maybe left aligned?

Reply:
Left aligned looks better IMO